core/pkg/ingress/annotations/rewrite/main.go

```diff
@@ -52,11 +52,7 @@ func NewParser(br resolver.DefaultBackend) parser.IngressAnnotation {
 // ParseAnnotations parses the annotations contained in the ingress
 // rule used to rewrite the defined paths
 func (a rewrite) Parse(ing *extensions.Ingress) (interface{}, error) {
-	rt, err := parser.GetStringAnnotation(rewriteTo, ing)
-	if err != nil {
-		return nil, err
-	}
-
+	rt, _ := parser.GetStringAnnotation(rewriteTo, ing)
```

Comment:
Yeah recommend returning errAnnotationNotFound or something and ignoring the error just based on the type, eg: 
```go
type errAnnotationMissing struct {
  errMsg string
}

func (e *errAnnotationMissing) Error() string { return e.errMsg}


rt, err := parser.GetStringAnnotation(rewriteTo, ing)
 switch t := err.(type) {
 case *errAnnotationMissing:
    break
 default:
     glog.Errorf("Failed to parse annotation, %v", err)
}
```